### PR TITLE
Knob to control

### LINKED ISSUE #18499: Update knobs to controls in storybook: ConfirmAddSuggestedToken
### Description
Currently there are a few storybook stories still using knobs in our storybook files `*.stories.js`. knobs have been deprecated in favour of controls. It would be great to migrate these last stories from knobs to controls.

This issue is to migrate the `ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js` from knobs to controls

Screenshot 2023-04-06 at 4 39 08 PM

### Requirements
If you would like to take on this issue please comment but before asking to take on this issue there are a few requirements that will help complete this task:
- Get storybook up and running `yarn storybook`
- Some knowledge of storybook is not required but would be beneficial for this task. [storybook docs here](https://storybook.js.org/)

### Technical Details
- Migrate knobs to controls

### Acceptance Criteria
- knobs have been migrated to controls for listed story
- Take Before / After screenshots of the knobs and new controls and attach to the PR
- Tag @georgewrmarshall, @garrettbear, or @NidhiKJha in the PR to review

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Explanation
1 .The component imports necessary dependencies, such as React, useEffect hook, and custom modules.
2. It configures the component as a story using Storybook's API. It sets the title, argTypes, and other settings for the story.
3 .The "PageSet" component is defined, which wraps around the "ConfirmAddSuggestedToken" component. It uses the useEffect hook to dispatch updates to the Redux store based on changes to the props passed to it.
4. The "DefaultStory" is defined as a story for the "ConfirmAddSuggestedToken" component. It uses the "PageSet" component and passes in default values for "suggestedAssets" and "tokens" props.
5 .Additional stories, "WithDuplicateAddress" and "WithDuplicateSymbolAndDifferentAddress", are defined with different prop values for "suggestedAssets" and "tokens" props, demonstrating different scenarios.

### Before

before

### After
Screenshot_2023-04-12_03-47-59 ( (1)
knob

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [] How reviewers can test my changes
- [] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [x] PR is linked to the appropriate GitHub issue
- [x] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
Fixes #18499

### CODE REVIEW

**Comment by sumitshinde-84** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by sumitshinde-84** [comment]
hii @cryptotavares  please review my PR .

**Comment by sumitshinde-84** [comment]
@danjm please review my PR, if there are any mistake please let me know

**Comment by sumitshinde-84** [comment]
@Gudahtt I have created this PR please review it

File: CHANGELOG.md [hunk 1]
```diff
@@ -3643,7 +3649,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
 - Added the ability to restore accounts from seed words.
 
-[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.28.1...HEAD
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.28.2...HEAD
+[10.28.2]: https://github.com/MetaMask/metamask-extension/compare/v10.28.1...v10.28.2
 [10.28.1]: https://github.com/MetaMask/metamask-extension/compare/v10.28.0...v10.28.1
 [10.28.0]: https://github.com/MetaMask/metamask-extension/compare/v10.27.0...v10.28.0
 [10.27.0]: https://github.com/MetaMask/metamask-extension/compare/v10.26.2...v10.27.0
```
Comment (georgewrmarshall) [inline CHANGELOG.md:3656]: This shouldn't be included in this PR. A bad merge from develop possibly?

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -2486,9 +2486,11 @@
...
     "description": "Description of a notification in the 'See What's New' popup. Describes Opensea Security Provider feature."
   },
+
   "notifications18Title": {
     "message": "Stay safe with security alerts"
   },
+
   "notifications19ActionText": {
     "message": "Enable NFT autodetection"
   },
```
Comment (georgewrmarshall) [inline app/_locales/en/messages.json:2496]: Changes to this file shouldn't be included in this PR. Please revert the changes

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "metamask-crx",
-  "version": "10.28.1",
+  "version": "10.28.2",
   "private": true,
   "repository": {
     "type": "git",
```
Comment (georgewrmarshall) [inline package.json:6]: Changes to this file should not be included in this PR. Please revert the changes

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 4]
```diff
@@ -18,15 +17,22 @@ export default {
...
+      control: 'text', // specify the control type as 'text'
+      defaultValue: 'META', // set the default value
+      table: { category: 'Data' },
+    },
+    image: {
+      control: 'text', // specify the control type as 'text'
+      defaultValue: 'metamark.svg', // set the default value
+      table: { category: 'Data' },
+    },
   },
```
Comment (georgewrmarshall) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:30]: `defaultValue` is deprecated please use default args
```suggestion
    symbol: {
      control: 'text',
      table: { category: 'Data' },
    },
    image: {
      control: 'text',
      table: { category: 'Data' },
    },
  },
  args: {
    symbol: 'ETH',
    image: './images/eth_logo.svg',
  },
```
I also think its probably best to make the default eth

**Review by georgewrmarshall — COMMENTED** [review]
Hey @sumitshinde-84, great start! It looks like some files got unintentionally changed possibly in a badge merge or rebase? Please remove the changes to all files except `ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js` otherwise it's looking pretty good. Have made one other suggestion
